designer/server/__tests__/view.tests.ts

```diff
@@ -60,4 +68,39 @@ suite("Server Help Pages", () => {
       ) > -1
     ).to.equal(true);
   });
+
+  test("Phase banner is present", async (flags) => {
+    const { persistenceService } = server.services();
+    persistenceService.listAllConfigurations = () => {
+      return Promise.resolve([]);
+    };
+
+    const options = {
+      method: "get",
+      url: "/new",
+    };
+
+    const res = await server.inject(options);
+    expect(res.statusCode).to.equal(200);
+    expect(
+      res.result.indexOf(
```

Comment:
Just a thought, but Cheerio can make this kind of thing easier (will parse the server text response into a queryable DOM object)

Reply:
Smoke tests are added by carlos , I am not sure how long will this banner last, don't want to bring in another dep